src/coffee/user_agent.coffee

```diff
@@ -15,6 +15,7 @@ UAs =
   android: /android/i
   ios: /(ipad|iphone|ipod)/i
   unix: /(linux|openbsd|freebsd|netbsd)/i
+  osx_catalina: /macintosh.+10_15/i
```

Comment:
It looks like this doesn't work as expected. I am on Catalina 10.15.7 and I am getting the list of apps for `osx` in general, not `osx_catalina` specifically:
![Screen Shot 2020-10-06 at 10 29 19](https://user-images.githubusercontent.com/7852553/95177894-09d47a80-07bf-11eb-8333-8a2a4685d855.png)

Reply:
That seems to be an issue with the userAgent string in Firefox, i updated the regex.

Reply:
It seems to work in Firefox and Chrome now. I can't test with Safari because by local build is served via HTTP and Safari blocks that, not sure how to serve my local build via HTTPS at the moment... but you should go ahead and deploy this branch to staging. Then we can test easily in all browsers, on all devices.

 I think the process would be:
- make a build,
- put it in the steady-cdn under `files/podigee_podcast_player/[date]-[git-sha]`
- commit that in steady-cdn, push to master, run the deploy script in that repo
- push a commit to main app staging that changes this URL: https://github.com/steady-media/main_app/blob/47988695fda72677550886310f81c214d99213a6/assets/js/components/podlove_subscribe_button.js#L9

It's a bit messy because we haven't done this very often...